packages/tcore-sdk/src/Types/Account/Account.types.test.ts

```diff
@@ -0,0 +1,128 @@
+import { ZodError } from "zod";
+import { zAccountList, zAccountCreate, zAccountTokenCreate, zAccountListQuery } from "./Account.types";
+
+describe("zAccountList", () => {
+  test("parses simple object", () => {
+    const data = [
+      {
+        password: "password",
+        id: "id",
+      },
+    ];
+    const parsed = zAccountList.parse(data);
+    expect(parsed[0].id).toEqual("id");
+  });
+
+  test("check required field", () => {
+    const data = [
+      {
+        password: "password",
+      },
+    ];
+    try {
+      zAccountList.parse(data);
+    } catch (error) {
+      const e = (error as ZodError).flatten();
+      expect(e.fieldErrors.id).toBe(undefined);
+    }
+  });
+
+  test("assure it is an array", () => {
+    const data = {
+      password: "password",
+      id: "id",
+    };
+    try {
+      zAccountList.parse(data);
+    } catch (error) {
+      const e = (error as ZodError).flatten();
+      expect(e.fieldErrors).toStrictEqual({});
+    }
+  });
+});
+
+describe("zAccountListQuery", () => {
+  test("parses simple object", () => {
+    const data = {
+      fields: ["id"],
+    };
+    const parsed = zAccountListQuery.parse(data);
+    expect(parsed.fields[0]).toEqual("id");
+  });
+
+  test("error if invalid option", () => {
+    const data = {
+      fields: [" "],
+    };
+    try {
+      zAccountListQuery.parse(data);
+    } catch (error) {
+      const e = (error as ZodError).flatten();
+      expect(e.fieldErrors.fields[0].startsWith("Invalid enum value.")).toBeTruthy();
+    }
+  });
+
+  test("assure empty query", () => {
+    const data = {
+      fields: [],
+    };
+    const parsed = zAccountListQuery.parse(data);
+    expect(parsed.fields).toBeNull;
```

Comment:
This doesn't make sense, this function is not being called. Please review everything like this and then i'll check the tests.